KitchenLib/src/BaseMod.cs

```diff
@@ -322,5 +345,19 @@ public KitchenLogger InitLogger()
 		{
 			return new KitchenLogger(ModName);
 		}
+
+		public Dictionary<string, CustomGameDataObject> RegisterGameDataObjects(Type t)
+		{
+			Dictionary<string, CustomGameDataObject> keyValuePairs = new Dictionary<string, CustomGameDataObject>();
+			Assembly asm = ModRegistery.keyValuePairs[t];
+			foreach (Type type in asm.GetTypes())
+			{
+				if(typeof(IAddGDO).IsAssignableFrom(type))
+				{
+					keyValuePairs.Add(type.Name, AddGameDataObject(type));
+				}
+			}
+			return keyValuePairs;
+		}

```

Comment:
Not sure if this is intentional, but this method doesn't actually register anything?